ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -170,12 +210,11 @@ export default class AmendRegistration extends Vue {
       this.emitError(financingStatement.error)
     } else {
       // load data into the store
-      const registrationType = RegistrationTypes.find(
-        (reg, index) => {
-          if (reg.registrationTypeAPI === financingStatement.type) {
-            return true
-          }
-        })
+      const registrationType = RegistrationTypes.find((reg, index) => {
```

Comment:
Just to clarify, we need the financing statement still, if we are loading a draft amendment?

Reply:
Yes, the way the api currently works a draft amendment only stores what is changing. The draft data (when complete) is identical to what is submitted in the registration request.